docs/resources/Guild.md

```diff
@@ -783,7 +783,8 @@ Returns the [guild](#DOCS_RESOURCES_GUILD/guild-object) object for the given id.
 
 ## Get Guild Preview % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/preview
 
-Returns the [guild preview](#DOCS_RESOURCES_GUILD/guild-preview-object) object for the given id. If the user is not in the guild, then the guild must be lurkable.
+Returns the [guild preview](#DOCS_RESOURCES_GUILD/guild-preview-object) object for the given id. 
+If the user is not in the guild, then the guild must have the "DISCOVERABLE" [feature](#DOCS_RESOURCES_GUILD/guild-object-guild-features).
```

Comment:
```suggestion
If the user is not in the guild, then the guild must be [discoverable](#DOCS_RESOURCES_GUILD/guild-object-guild-features).
```
Understand where your edit is coming from, but want to stick with saying a guild is discoverable, rather than has the "DISCOVERABLE" feature.